int64 id
string name
geometry_msgs/Pose pose
